src/tiled/mapdocument.h

```diff
@@ -492,6 +494,15 @@ inline void MapDocument::emitTileLayerDrawMarginsChanged(TileLayer *layer)
 }
 
 /**
+ * Emits the signal notifying about the xy-offset of a tile changing.
+ */
+inline void MapDocument::emitTileOffsetXYChanged(Tile *tile)
+{
+    emit tileOffsetXYChanged(tile);
+    emit mapChanged();
```

Comment:
Please leave this second signal out if you can. Instead, you should connect to `tileOffsetXYChanged` (to be renamed to `tileOffsetChanged` in relevant places.